app/authenticated/account/route.js

```diff
@@ -0,0 +1,20 @@
+import Ember from 'ember';
+import CurrentUserMixin from 'appy-ui/mixins/current-user';
+
+const { Route, get } = Ember;
+
+export default Route.extend(CurrentUserMixin, {
+	model () {
+		return get(this, 'currentUser');
```

Comment:
Internally, `this.get` is nothing more than a call to `Ember.get`, so calling the latter directly removes a method call from the stack.

The performance gain from this will almost always be unnoticeable, but `Ember.get` also has the advantage of working with POJOs, which don't (necessarily) have a `.get` method. For example, in a component that accepts a `data` attribute, the outside world might decide it's more convenient to pass a POJO than an `Ember.Object`-derived type. If the component attempts to do `data.get('someProp')`, it will either throw if `data` doesn't have a `.get` method defined, or call a method it didn't expect. If it does `Ember.get(data, 'someProp'), it'll carry on happily, avoiding that issue..

You could mix and match between the two approaches depending on the context, but I like to keep my code consistent, and given that the `Ember.get` approach has the two advantages above, that's become my preferred choice.